students/Daniel_Spray/Lesson03/mailroom.py

```diff
@@ -0,0 +1,106 @@
+def main(b=(["William Gates, III","Mark Zuckerberg","Jeff Bezos","Paul Allen"],[653784.49,16396.10,877.33,708.42], [2,3,1,3], [], [])):
+    print("This program will hopefully help you send some meaningful messages")
+    print("Type the corresponding number to select from the following list:")
+    print(" ")
+    print("1: Send a Thank You")
+    print("2: Create a Report")
+    print("3: Quit")
+    a=input(">")
+	
+    if str(a) == "1":
+        saty()
+    elif str(a) == "2":
+        car(calculation(b[0],b[1],b[2],b[3],b[4]))
+    elif str(a) == "3":
+        something="nothing"
+    else:
+        print("Sorry, I didn't recognize that command")
+        main()
+
+def saty():
+    
+    Donors = ["William Gates, III","Mark Zuckerberg","Jeff Bezos","Paul Allen"]
```

Comment:
you should not be hardcoding the names here...

also var names should be lower case